requirements.txt

```diff
@@ -1 +1,3 @@
-neuraxle>=0.3.1
+neuraxle>=0.5.2
+scipy==1.4.1
+tensorflow
```

Comment:
I would remove tensorflow here. Because if people prefer to install `tensorflow-gpu` it may conflict. So they should install tensorflow manually and remove it from requirements. 

Was scipy really needed ? Do we import scipy somewhere ? I dont think so. Let's keep requirements lightweigjt.